src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#include <ray/gcs/redis_gcs_client.h>
+#include <ray/rpc/gcs_server/job_info_access_server.h>
+#include <ray/rpc/grpc_server.h>
+#include <string>
+
+namespace ray {
+namespace gcs {
+
+struct GcsServerConfig {
+  std::string server_name = "GcsServer";
+  uint16_t server_port = 0;
+  uint16_t server_thread_num = 1;
```

Comment:
What is this thread number?

Reply:
It is the input argument of GrpcServer.

Reply:
I mean, why do we need to config this? For now, `GrpcServer` is single threaded with an event loop. In the future, if we want to make it multithreading, we can add this config later.

Reply:
oh, I misunderstood the purpose. What about renaming the above 3 configs as `grpc_server_name`, `grpc_server_port`, etc?

Reply:
> oh, I misunderstood the purpose. What about renaming the above 3 configs as `grpc_server_name`, `grpc_server_port`, etc?

Okey I will fix it soon.